### DISCUSSION TITLE
Audio improvements master topic

### DISCUSSION THREAD

**Discussion by xezon** [TheSuperHackers/GeneralsGamePatch#684] [General]
Audio improvements master topic. Are there any sounds that need improvement because the original is bad or broken? Audio preferences are very subjective, so swapping audio files needs to be carefully evaluated. When in doubt, prefer to not change Audio.

See also:
* #176

**Comment by Haubibban** [comment]
New A-10 Minigun sounds.

From:
Zero Hour - Specter Gunship Minigun -
[ZH_A-10.zip](https://github.com/xezon/GeneralsGamePatch/files/7265340/ZH_A-10.zip)

To:
ROTR - Realistic BRRRRRT Minigun -
[ROTR_A-10.zip](https://github.com/xezon/GeneralsGamePatch/files/7265341/ROTR_A-10.zip)

**Comment by xezon** [comment]
A10 sound change is a matter of taste. Sounds very different.

**Comment by commy2** [comment]
Burton has a different rifle sound in CCG vs ZH. I personally liked the CCG one more.
